tests/functional/object/mcg/test_noobaa_priority_class.py

```
@@ -225,6 +227,129 @@ def _verify_pod_priority_classes(self, namespace, core_pc, db_pc, endpoint_pc):
+        2. Wait for operator reconciliation; restart db pods
+        3. Verify all NooBaa pods have the correct PriorityClassName
+        4. Record the creation times of all NooBaa core, db, and endpoint pods
+        5. Restart the NooBaa operator pod
+        6. Wait for the NooBaa operator pod to be ready
+        7. Verify all NooBaa pods still have the correct PriorityClassName
+        8. Verify no pods were unnecessarily restarted (creation times unchanged)
+        """
+        namespace = config.ENV_DATA["cluster_namespace"]
+        noobaa_ocp = OCP(kind=constants.NOOBAA_RESOURCE_NAME, namespace=namespace)
+
+        def finalizer():
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)